docs/docsite/rst/conf.py

```diff
@@ -27,6 +27,7 @@
 #
 sys.path.insert(0, os.path.join('ansible', 'lib'))
 sys.path.append(os.path.abspath('_themes'))
```

Comment:
I'm not sure whether this works (or is needed at all), since `_themes` in one level up (as the new `extensions` is as well).

Reply:
Any further thoughts on whether this is needed?

Reply:
No idea. I've now simply removed it; if something breaks (which I don't think should happen), I guess we'll notice quickly enough :-)